lib/index.js

```diff
@@ -1972,7 +1986,15 @@ exposeMethod('filter');
  * @param {Function} f - the truth test function which returns a Stream
  * @api public
  *
- * var checkExists = _.wrapCallback(fs.exists);
+ * function checkExists(file) {
+ *     return _(function (push, next) {
+ *         fs.exists(file, function (exists) {
```

Comment:
I thought about just changing it to `fs.access`, but the other side of that argument is, "should we be using a function that doesn't exist in 0.10 when we technically support it?"

That said, Node 4 is the LTS version, so maybe not many people use 0.10 anymore. I don't feel strongly about it though, so if you don't think it matters, then I'm happy to change it.